mac

```diff
@@ -4,13 +4,42 @@
 # Be prepared to turn your laptop (or desktop, no haters here)
 # into an awesome development machine.
 
+show_help() {
```

Comment:
After running the script on my machine, now I have this error with foreman 😞 

![image](https://user-images.githubusercontent.com/9013072/138166177-58b2b2de-1611-413e-9d4c-13ab404ea1a3.png)